packages/interaction-kit/src/structures/embed.ts

```diff
@@ -125,6 +134,7 @@ export default class Embed {
 	}
 
 	serialize(): APIEmbed {
+		// @ts-expect-error it really wants me to do this as optional args
```

Comment:
I should probably fix this? It seems very unnecessary though

Reply:
Change the return type to some type of post body ['embeds'][0] which will have this fancy shindig applied to it: `AddUndefinedToPossiblyUndefinedPropertiesOfInterface`